components/schedule-editor/src/ScheduleEditor.svelte

```diff
@@ -166,16 +166,22 @@
 
   // Properties requiring further manipulation:
   let startDate: string = new Date().toLocaleDateString("en-CA");
+  let startDateMethod: "date" | "current" = "current";
 
   function transformPropertyValues() {
-    startDate = _this.start_date?.toLocaleDateString("en-CA");
+    startDate = _this.start_date?.toLocaleDateString("en-CA") as string;
+    startDateMethod = _this.start_date ? "date" : "current";
   }
 
-  $: {
-    _this.start_date = new Date(
-      new Date(startDate).getTime() -
-        new Date(startDate).getTimezoneOffset() * -60000,
-    );
+  $: if (initialized) {
+    if (startDateMethod === "date") {
```

Comment:
Why is startDateMethod needed here? Can't we just check `if (_this.start_date)`?

Reply:
Hm, I think I tried this, but found that there was a limitation with the value being recorded as default before the manifest is loaded, and setting the start_date to null as a result, thus overriding the eventual manifest entry.

This may have been encountered before the `if (initialized)` check though. Will try it again.